apps/laboratory/src/components/Wagmi/WagmiPurchaseDonutSyncPermissionsTest.tsx

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable multiline-comment-style */
+/* eslint-disable capitalized-comments */
```

Comment:
we should remove this after cleaning up the comment

Reply:
removed